interface/usergroup/user_info_ajax.php

```diff
@@ -45,8 +36,8 @@
 $errMsg='';
 $success=update_password($_SESSION['authId'], $_SESSION['authId'], $curPass, $newPass, $errMsg);
 if ($success) {
-    echo xlt("Password change successful");
+    echo "<div class='alert alert-success'>" . xlt("Password change successful") . "</div>";
```

Comment:
@robertdown ,
btw, here's a link to the bootstrap alert classes:
https://www.w3schools.com/bootstrap/bootstrap_alerts.asp